kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableReadsSuite.scala

```diff
@@ -491,6 +491,15 @@ class DeltaTableReadsSuite extends AnyFunSuite with TestUtils {
     )
   }
 
+  test("table with v2 checkpoint") {
+    Seq("json", "parquet").foreach { fileFormat =>
+      checkTable(
+        path = goldenTablePath(s"clustered-checkpoint-v2-$fileFormat"),
```

Comment:
is this a duplicate of the test added in `LogReplaySuite`?

Reply:
Also can you create a new suite `CheckpointV2ReadSuite.scala` and move the tests there?

Reply:
Yes, this is the same.